type/src/ai-card.ts

```diff
@@ -0,0 +1,197 @@
+// --8<-- [start:AICard]
+/**
+ * The unified AI Card object for a single agent.
+ */
+export interface AICard {
+  /**
+   * Declares the schema this card adheres to.
+   * e.g., "https://ai-agent-protocol.org/ai-card/v1/schema.json"
+   */
+  $schema: string;
+
+  /**
+   * The version of the AI Card specification itself.
+   */
+  specVersion: string;
+  
+  /**
+   * The primary verifiable ID for the agent (e.g., DID).
+   */
+  id: string; 
```

Comment:
Is it a requirement that this is "verifiable" or would that just be a recommendation?